jax/_src/lax/parallel.py

```diff
@@ -249,6 +249,28 @@ def _canonicalize_axis_index_groups(axis_index_groups):
     return
   return tuple(map(tuple, axis_index_groups))
 
+
+def pcollective_broadcast(x, axis_name, source):
```

Comment:
I would've preferred this to be called `pbroadcast`, but that was already taken. Which leads me to the question: Is the current `pbroadcast` actually the same as this `pcollective_broadcast`?

Reply:
Answer is no, pbroadcast doesn't tke in a source or root index. https://github.com/google/jax/blob/b9824d7de3cb30f1df738cc42e486db3e9d915ff/jax/experimental/shard_map.py#L839